flask_server.py

```diff
@@ -171,11 +125,46 @@ def gallery_api(username, date):
         to_render_items = get_folders(user_path)
     else:
         to_render_items = get_folders(folder_path)
-    return jsonify({'items' : to_render_items})
+    return jsonify({'items': to_render_items})
+
+@app.route("/api/settings/", methods=['GET', 'POST'])
+def get_settings_api():
+    if request.method == 'GET':
+        logger.info(f"API/GET request to /settings/")
```

Comment:
I'd move the logger call above and handle both the POST and the GET like we do at line 98